modules/llvm/17.0.2/presubmit.yml

```diff
@@ -0,0 +1,13 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    build_targets:
+    - '@llvm//:Core'
```

Comment:
It would be nice to run at least _one_ test target. Maybe `@llvm//unittests:adt_tests`? It would be _really_ nice to run all the unit tests to be honest (`@llvm//unittests:all`).